src/features/todos/ToDoForm.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addToDo } from './ToDoSlice'
+
+const ToDoForm = () => {
+    const [toDo, setTodo] = useState([])
```

Comment:
Since this is the users input it would be for it just to be an empty string rather than an empty object.